website/templates/profile/account.mako

```diff
@@ -65,8 +65,12 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
-                                            <p>To merge an existing account with this one or to log in with multiple email addresses, add an alternate email address below.</p>
-
+                                            <p>
+                                            To merge an existing account with this one or to log in with multiple email addresses, add an alternate email address below.
+                                            <a id="addDashboardFolder" class="fa fa-info-circle social-icons" data-bind="tooltip: {title: 'Merging accounts will move all projects and components associated with two emails into one account. All projects and components will be displayed under the email address listed as primary.',
+                                             placement: 'bottom', container : 'body'}" class="btn btn-default"></a>
```

Comment:
The `class` attribute is listed twice in this element -- I think you can just remove `class="btn btn-default` since the element is not a button.